File: docs/components_page/components/card/outline.R
library(dashBootstrapComponents)

card_content <- list(
  dbcCardHeader("Card header"),
  dbcCardBody(
    list(
      html$h5("Card title", className = "card-title"),
      p(
        "This is some card content that we'll reuse",
        className = "card-text"
      )
    )
  )
)

row_1 <- dbcRow(
  list(
    dbcCol(dbcCard(card_content, color = "primary", outline = TRUE)),
    dbcCol(dbcCard(card_content, color = "secondary", outline = TRUE)),
    dbcCol(dbcCard(card_content, color = "info", outline = TRUE))
  ),
  className = "mb-4"
)

row_2 <- dbcRow(
  list(
    dbcCol(dbcCard(card_content, color = "success", outline = TRUE)),
    dbcCol(dbcCard(card_content, color = "warning", outline = TRUE)),
    dbcCol(dbcCard(card_content, color = "danger", outline = TRUE))
  ),
  className = "mb-4"
)

row_3 <- dbcRow(
  list(
    dbcCol(dbcCard(card_content, color = "light", outline = TRUE)),
    dbcCol(dbcCard(card_content, color = "dark", outline = TRUE))
  )
)

cards <- div(list(row_1, row_2, row_3))

File: docs/components_page/components/card/outline.jl
using DashBootstrapComponents

card_content = [
    dbc_cardheader("Card header"),
    dbc_cardbody([
        html_h5("Card title", className = "card-title"),
        html_p("This is some card content that we'll reuse", className = "card-text"),
    ]),
];

row_1 = dbc_row(
    [
        dbc_col(dbc_card(card_content, color = "primary", outline = true)),
        dbc_col(dbc_card(card_content, color = "secondary", outline = true)),
        dbc_col(dbc_card(card_content, color = "info", outline = true)),
    ],
    className = "mb-4",
);

row_2 = dbc_row(
    [
        dbc_col(dbc_card(card_content, color = "success", outline = true)),
        dbc_col(dbc_card(card_content, color = "warning", outline = true)),
        dbc_col(dbc_card(card_content, color = "danger", outline = true)),
    ],
    className = "mb-4",
);

row_3 = dbc_row([
    dbc_col(dbc_card(card_content, color = "light", outline = true)),
    dbc_col(dbc_card(card_content, color = "dark", outline = true)),
]);

cards = html_div([row_1, row_2, row_3]);
